src/runtime-rs/crates/resource/src/rootfs/mod.rs

```diff
@@ -59,60 +59,21 @@ impl RootFsResource {
         hypervisor: &dyn Hypervisor,
         sid: &str,
         cid: &str,
-        root: &oci::Root,
+        root_path: &str,
         bundle_path: &str,
         rootfs_mounts: &[Mount],
-    ) -> Result<Arc<dyn Rootfs>> {
+        is_image_offload: bool,
+    ) -> Result<Option<Arc<dyn Rootfs>>> {
         match rootfs_mounts {
-            // if rootfs_mounts is empty
             mounts_vec if mounts_vec.is_empty() => {
```

Comment:
I think this change may cause it doesn't work for `virt-container`, maybe consider another way to make it work for both.

Reply:
For which case, `virt_container` will not work?